UnleashClient/__init__.py

```diff
@@ -211,12 +214,19 @@ def initialize_client(self, fetch_toggles: bool = True) -> None:
         if not self.is_initialized:
             # pylint: disable=no-else-raise
             try:
+                headers = {  # SDK identification headers
+                    **self.unleash_custom_headers,
+                    "x-unleash-connection-id": self.unleash_connection_id,
+                    "x-unleash-appname": self.unleash_app_name,
+                    "x-unleash-sdk": f"{SDK_NAME}:{SDK_VERSION}",
+                }
+
                 # Setup
                 metrics_args = {
                     "url": self.unleash_url,
                     "app_name": self.unleash_app_name,
                     "instance_id": self.unleash_instance_id,
-                    "custom_headers": self.unleash_custom_headers,
+                    "custom_headers": headers,
```

Comment:
I think it's okay to pass this like this but this does now mean that this isn't just custom headers and the places where the callees are receiving this are now misleading

Any chance we can rename those call sites to "headers" or something clearer?

Reply:
Like so: https://github.com/Unleash/unleash-client-python/pull/342/commits/fc4685f3b82ecd50316d752fbfd1dbeca5a5ef25 ?